bbc/simorgh
# WSTEAM1-329 - Delete legacy colours from components, layouts, utilities & pages

Resolves JIRA WSTEAM1-329

Overall changes
======
Remove references to psammead legacy colours within src/app/legacy/components, src/app/Layouts, src/app/lib/utilities, src/app/pages.

Code changes
======
- Update legacy colour references in rc/app/legacy/components, src/app/Layouts, src/app/lib/utilities, src/app/pages to use ThemeProvider equivalents.
- Remove all imports from psammead/psammead-styles/src/colours.
- Most instances of colour references are inside styled components. There are some exceptions. For these I have used the useTheme() hook and direct imports from ThemeProvider/palette.
- Update failing unit tests. In most instances this means updating the render() function import to be from react-testing-library-with-providers, and refactoring snapshot tests to use the .toMatchSnapshot() method.
- Fix broken .stories files to by importing and wrapping component in ThemeProvider

Questions
======
- I have added some Questions I have as comments on this PR.

Testing
======
- Check all legacy colour imports have been removed by pulling down the `WS-329-delete-legacy-colours-components` branch and search` '/src/colours';` for within the above file paths, e.g. src/app/legacy/components/*
- Run unit tests
- Check storybook components

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
